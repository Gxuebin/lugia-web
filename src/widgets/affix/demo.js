/**
 *
 * create by guorg
 *
 * @flow
 */
import * as React from 'react';
import styled from 'styled-components';
import Affix from './affix';
import Button from '../button';

export default class AffixDemo extends React.Component {
  constructor() {
    super();
    this.EleRef = null;
    this.Ele2Ref = null;
    this.state = {
      display: true,
    };
  }
  render() {
    const { display } = this.state;
    return (
      <div>
        {display ? (
          <div style={{ width: '200px', height: '1200px' }}>
            <div style={{ width: '200px', height: '200px' }} />
            <Affix offsetTop={50}>
              <Button>affix-top</Button>
            </Affix>

            <div
              style={{ width: '200px', height: '200px', overflowY: 'scroll' }}
              ref={node => (this.EleRef = node)}
            >
              <div style={{ height: '400px' }}>
                <div style={{ width: '20px', height: '100px' }} />
                <Affix offsetTop={50} target={() => this.EleRef}>
                  <Button>affix-top</Button>
                </Affix>
              </div>
            </div>
            <div
              style={{ width: '200px', height: '200px', overflowY: 'scroll' }}
              ref={node => (this.Ele2Ref = node)}
            >
              <div style={{ height: '400px' }}>
                <div style={{ width: '20px', height: '200px' }} />
                <Affix offsetBottom={50} target={() => this.Ele2Ref}>
                  <Button>affix2-bottom</Button>
                </Affix>
              </div>
            </div>

            <div style={{ width: '200px', height: '500px' }} />
            <Affix offsetBottom={50}>
              <Button>affix-bottom</Button>
            </Affix>
            <div style={{ width: '200px', height: '500px' }} />
          </div>
        ) : (
          <div style={{ width: '200px', height: '1200px' }} />
        )}
      </div>
    );
  }
}
