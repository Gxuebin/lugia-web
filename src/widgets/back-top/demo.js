/**
 *
 * create by guorg
 *
 * @flow
 */
import * as React from 'react';
import styled from 'styled-components';
import BackTop from './back-top';
import Widget from '../consts/index';
import Theme from '../theme';

const Demo = styled.div`
  & > span > div > div {
    bottom: ${props => 50 + props.cur * 50}px;
  }
`;
const DemoBack = styled.div`
  width: 40px;
  height: 40px;
  line-height: 40px;
  text-align: center;
  border-radius: 4px;
  border: 1px solid #e8e8e8;
`;
export default class AffixDemo extends React.Component<any, any> {
  EleRef: any;
  render() {
    const view = {
      [Widget.BackTop]: {
        normal: {
          color: 'red',
          background: { backgroundColor: 'orange' },
          width: 100,
          height: 100,
          borderRadius: 100,
          opacity: 0.7,
          border: {
            borderWidth: 1,
            borderStyle: 'solid',
            borderColor: 'red',
          },
        },
      },
    };
    return (
      <div style={{ height: '1600px' }}>
        <BackTop />
        <Demo cur={1}>
          <BackTop>
            <DemoBack>UP</DemoBack>
          </BackTop>
        </Demo>
        <Demo cur={2}>
          <Theme config={view}>
            <BackTop />
          </Theme>
        </Demo>
        <div
          style={{ width: '200px', height: '200px', overflowY: 'scroll', marginTop: '400px' }}
          ref={node => (this.EleRef = node)}
        >
          <Demo cur={3}>
            <div style={{ width: '20px', height: '300px' }} />
            <BackTop visibilityHeight={80} target={() => this.EleRef} />
          </Demo>
        </div>
      </div>
    );
  }
}
