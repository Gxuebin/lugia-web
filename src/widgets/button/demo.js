/**
 *
 * create by ligx
 *
 * @flow
 */
import React from 'react';
import Button from './index';

export default () => {
  return <Button type="danger">hello</Button>;
};
