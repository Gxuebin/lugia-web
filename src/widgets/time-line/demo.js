/**
 *
 * create by liangguodong on 2018/11/19
 *
 * @flow
 */
import * as React from 'react';
import TimeLine from './timeLine';
import TimeLineItem from './timeLineItem';
import Theme from '../theme/';
import Loading from '../loading/';
import styled from 'styled-components';

class TimeLinePending extends React.Component<Object, Object> {
  state = {
    reverse: false,
  };

  handleClick = () => {
    this.setState({ reverse: !this.state.reverse });
  };

  render() {
    return (
      <div style={{ margin: 30 }}>
        <button type="primary" onClick={this.handleClick}>
          点击反转
        </button>
        <TimeLine reverse={this.state.reverse}>
          <TimeLineItem time="2018-01-01" description={'description111'} />
          <TimeLineItem time="2018-01-02" description={'description222'} />
          <TimeLineItem time="2018-01-03" description={'description333'} />
          <TimeLineItem time="2018-01-04" description={'description444'} />
          <TimeLineItem time="2018-01-05" description={'description555'} />
        </TimeLine>
      </div>
    );
  }
}

const Wrapper = styled.div`
  text-align: left;
  margin: 100px;
  display: inline-block;
`;
const view = {
  long: {
    height: 100,
  },
  short: {
    height: 20,
  },
};
export const SimpleDemo = () => {
  return (
    <Theme config={view}>
      <div>
        <Wrapper>
          <p>simple</p>
          <br />
          <TimeLine>
            <TimeLineItem viewClass={'long'} time="2018-01-01" />
            <TimeLineItem time="2018-01-02" />
            <TimeLineItem time="2018-01-03" />
            <TimeLineItem viewClass={'short'} time="2018-01-04" />
            <TimeLineItem time="2018-01-05" />
          </TimeLine>
        </Wrapper>
        <Wrapper>
          <p>mode alternate</p>
          <br />
          <TimeLine mode={'alternate'}>
            <TimeLineItem time="2018-01-01" />
            <TimeLineItem time="2018-01-02" />
            <TimeLineItem time="2018-01-03" />
            <TimeLineItem time="2018-01-04" />
            <TimeLineItem time="2018-01-05" />
          </TimeLine>
        </Wrapper>
        <Wrapper>
          <p>正常</p>
          <br />
          <TimeLine>
            <TimeLineItem time="2018-01-01" description={'description111'} />
            <TimeLineItem time="2018-01-02" description={'description222'} />
            <TimeLineItem time="2018-01-03" description={'description333'} />
            <TimeLineItem time="2018-01-04" description={'description444'} />
            <TimeLineItem time="2018-01-05" description={'description555'} />
          </TimeLine>
        </Wrapper>
        <Wrapper>
          <p>mode:alternate </p>
          <br />
          <TimeLine mode={'alternate'}>
            <TimeLineItem time="2018-01-01" description={'description111'} />
            <TimeLineItem time="2018-01-02" description={'description222'} />
            <TimeLineItem time="2018-01-03" description={'description333'} />
            <TimeLineItem time="2018-01-04" description={'description444'} />
            <TimeLineItem time="2018-01-05" description={'description555'} />
          </TimeLine>
        </Wrapper>
        <Wrapper>
          <p>不同状态</p>
          <br />
          <TimeLine direction={'left'}>
            <TimeLineItem time="2018-01-01" status={'success'} />
            <TimeLineItem time="2018-01-02" />
            <TimeLineItem time="2018-01-03" status={'failed'} />
            <TimeLineItem time="2018-01-04" />
            <TimeLineItem time="2018-01-05" />
          </TimeLine>
        </Wrapper>
        <Wrapper>
          <p>type icon </p>
          <br />
          <TimeLine>
            <TimeLineItem
              time="2018-01-01"
              description={'description111'}
              type={'icon'}
              icon={'lugia-icon-financial_archive'}
            />
            <TimeLineItem time="2018-01-02" description={'description222'} />
            <TimeLineItem time="2018-01-03" description={'description333'} />
            <TimeLineItem
              time="2018-01-04"
              description={'description444'}
              type={'icon'}
              icon={'lugia-icon-financial_archive'}
            />
            <TimeLineItem time="2018-01-05" description={'description555'} />
          </TimeLine>
        </Wrapper>
        <Wrapper>
          <p>pending </p>
          <br />
          <TimeLine pending={true}>
            <TimeLineItem time="2018-01-01" description={'description111'} />
            <TimeLineItem time="2018-01-02" description={'description222'} />
            <TimeLineItem time="2018-01-03" description={'description333'} />
            <TimeLineItem time="2018-01-04" description={'description444'} />
            <TimeLineItem time="2018-01-05" description={'description555'} />
          </TimeLine>
        </Wrapper>
        <TimeLinePending />,
      </div>
    </Theme>
  );
};
export const OtherDemo = () => {
  return (
    <div>
      <Wrapper>
        <p>pending </p>
        <br />
        <TimeLine>
          <TimeLineItem time="2018-01-01" description={'description111'} />
          <TimeLineItem time="2018-01-02" description={'description222'} type="explain" />
          <TimeLineItem time="2018-01-03" description={'description333'} />
          <TimeLineItem time="2018-01-04" description={'description444'} type="explain" />
          <TimeLineItem time="2018-01-05" description={'description555'} />
        </TimeLine>
      </Wrapper>
      <Wrapper>
        <p>pending </p>
        <br />
        <TimeLine>
          <TimeLineItem time="2018-01-01" />
          <TimeLineItem time="2018-01-02" type="explain" />
          <TimeLineItem time="2018-01-03" type="explain" />
          <TimeLineItem time="2018-01-04" type="explain" />
          <TimeLineItem time="2018-01-05" />
        </TimeLine>
      </Wrapper>
    </div>
  );
};
export default () => {
  return [<SimpleDemo />, <OtherDemo />];
};
