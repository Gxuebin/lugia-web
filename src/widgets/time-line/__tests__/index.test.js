/**
 *
 * create by lianggd
 *
 *
 */
import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import { SimpleDemo } from '../demo';

describe('timeLineDemo', () => {
  it('Component JSON', () => {
    const Target = <SimpleDemo />;
    expect(renderer.create(Target).toJSON()).toMatchSnapshot();
  });
});
