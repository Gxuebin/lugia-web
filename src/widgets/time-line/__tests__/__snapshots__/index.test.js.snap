// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`timeLineDemo Component JSON 1`] = `
.c21 {
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  cursor: pointer;
  font-size: 1.2rem;
}

.c2 {
  position: relative;
  height: 8.333333333333334em;
  width: 100%;
}

.c8 {
  position: relative;
  height: 3.5000000000000004em;
  width: 100%;
}

.c10 {
  position: relative;
  height: 1.6666666666666667em;
  width: 100%;
}

.c13 {
  position: relative;
  height: 4.500000000000001em;
  width: 100%;
}

.c7 {
  text-align: inherit;
  font-size: 1.4rem;
  max-width: 12.5em;
  overflow: hidden;
  white-space: nowrap;
  color: #333333;
}

.c15 {
  text-align: inherit;
  font-size: 1.4rem;
  max-width: 12.5em;
  overflow: hidden;
  white-space: nowrap;
  line-height: 1.5;
  color: #666666;
}

.c6 {
  position: absolute;
  top: 0;
  text-align: left;
  left: 0;
  margin-left: 2.0833333333333335em;
}

.c17 {
  position: absolute;
  top: 0;
  text-align: right;
  right: 0;
  margin-right: 2.0833333333333335em;
}

.c3 {
  position: relative;
  width: 1.6666666666666667em;
  text-align: center;
}

.c5 {
  position: relative;
  left: 0.8333333333333334em;
  height: 8.333333333333334em;
  border-left: 0.08333333333333334em solid #e8e8e8;
  z-index: 2;
}

.c9 {
  position: relative;
  left: 0.8333333333333334em;
  height: 100%;
  border-left: 0.08333333333333334em solid #e8e8e8;
  z-index: 2;
}

.c11 {
  position: relative;
  left: 0.8333333333333334em;
  height: 1.6666666666666667em;
  border-left: 0.08333333333333334em solid #e8e8e8;
  z-index: 2;
}

.c12 {
  position: relative;
  left: 0.8333333333333334em;
  height: 100%;
  border-left: 0.08333333333333334em solid #e8e8e8;
  display: none;
  z-index: 2;
}

.c14 {
  position: relative;
  left: 0.8333333333333334em;
  height: inherit;
  border-left: 0.08333333333333334em solid #e8e8e8;
  z-index: 2;
}

.c16 {
  position: relative;
  left: 0.8333333333333334em;
  height: inherit;
  border-left: 0.08333333333333334em solid #e8e8e8;
  display: none;
  z-index: 2;
}

.c4 {
  position: absolute;
  background: #4d63ff;
  width: 0.8333333333333334em;
  height: 0.8333333333333334em;
  left: 0.45833333333333337em;
  z-index: 3;
  border-radius: 50%;
  -webkit-transition: all 0.3s linear 0.1s;
  transition: all 0.3s linear 0.1s;
}

.c18 {
  position: absolute;
  background: #56c22d;
  width: 0.8333333333333334em;
  height: 0.8333333333333334em;
  left: 0.45833333333333337em;
  z-index: 3;
  border-radius: 50%;
  -webkit-transition: all 0.3s linear 0.1s;
  transition: all 0.3s linear 0.1s;
}

.c19 {
  position: absolute;
  background: #f22735;
  width: 0.8333333333333334em;
  height: 0.8333333333333334em;
  left: 0.45833333333333337em;
  z-index: 3;
  border-radius: 50%;
  -webkit-transition: all 0.3s linear 0.1s;
  transition: all 0.3s linear 0.1s;
}

.c20 {
  position: absolute;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  text-align: center;
  font-size: 1.4rem;
  background: #ffffff;
  z-index: 1;
  left: 0.2916666666666667em;
  -webkit-animation: rotate 1s linear infinite;
  animation: rotate 1s linear infinite;
}

.c1 {
  height: 25em;
  width: 1.6666666666666667em;
}

.c22 {
  display: inline-block;
  margin-bottom: 0;
  box-sizing: border-box;
  text-align: center;
  touch-action: manipulation;
  cursor: pointer;
  white-space: nowrap;
  line-height: 1;
  font-family: Trebuchet Ms,Arial,Helvetica,sans-serif;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  -webkit-transition: all 0.3s cubic-bezier(0.645,0.045,0.355,1);
  transition: all 0.3s cubic-bezier(0.645,0.045,0.355,1);
  position: relative;
  text-transform: none;
  outline: 0;
  color: #ffffff;
  background-color: #4d63ff;
  border: none;
  padding: 0 1.2857142857142858em;
  font-size: 1.4rem;
  border-radius: 0.28571428571428575em;
}

.c22:hover {
  color: #ffffff;
  border: none;
  background-color: #8090ff;
}

.c22:focus {
  color: #ffffff;
  border: none;
  background-color: #8090ff;
}

.c22:active {
  background-color: #3e4fcc;
}

.c23 {
  display: inline-block;
  line-height: 2.285714285714286em;
}

.c0 {
  text-align: left;
  margin-top: 50px;
  margin-left: 100px;
  display: inline-block;
}

<div>
  <div
    className="c0"
  >
    <p>
       调节高度的 简洁样式
    </p>
    <br />
    <div
      className="c1"
    >
      <div
        className="c2"
        type={[Function]}
      >
        <div
          className="c3"
        >
          <div
            className="c4"
            type={[Function]}
          />
        </div>
        <div
          className="c5"
          type={[Function]}
        />
        <div
          className="c6"
          direction="right"
        >
          <div
            className="c7"
          >
            2018-01-01
             
          </div>
        </div>
      </div>
      <div
        className="c8"
        type={[Function]}
      >
        <div
          className="c3"
        >
          <div
            className="c4"
            type={[Function]}
          />
        </div>
        <div
          className="c9"
          type={[Function]}
        />
        <div
          className="c6"
          direction="right"
        >
          <div
            className="c7"
          >
            2018-01-02
             
          </div>
        </div>
      </div>
      <div
        className="c8"
        type={[Function]}
      >
        <div
          className="c3"
        >
          <div
            className="c4"
            type={[Function]}
          />
        </div>
        <div
          className="c9"
          type={[Function]}
        />
        <div
          className="c6"
          direction="right"
        >
          <div
            className="c7"
          >
            2018-01-03
             
          </div>
        </div>
      </div>
      <div
        className="c10"
        type={[Function]}
      >
        <div
          className="c3"
        >
          <div
            className="c4"
            type={[Function]}
          />
        </div>
        <div
          className="c11"
          type={[Function]}
        />
        <div
          className="c6"
          direction="right"
        >
          <div
            className="c7"
          >
            2018-01-04
             
          </div>
        </div>
      </div>
      <div
        className="c8"
        type={[Function]}
      >
        <div
          className="c3"
        >
          <div
            className="c4"
            type={[Function]}
          />
        </div>
        <div
          className="c12"
          type={[Function]}
        />
        <div
          className="c6"
          direction="right"
        >
          <div
            className="c7"
          >
            2018-01-05
             
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="c0"
  >
    <p>
      有详情描述 简洁样式 
    </p>
    <br />
    <div
      className="c1"
    >
      <div
        className="c13"
        type={[Function]}
      >
        <div
          className="c3"
        >
          <div
            className="c4"
            type={[Function]}
          />
        </div>
        <div
          className="c14"
          type={[Function]}
        />
        <div
          className="c6"
          direction="right"
        >
          <div
            className="c7"
          >
            2018-01-01
             
          </div>
          <div
            className="c15"
          >
            description111
             
          </div>
        </div>
      </div>
      <div
        className="c13"
        type={[Function]}
      >
        <div
          className="c3"
        >
          <div
            className="c4"
            type={[Function]}
          />
        </div>
        <div
          className="c14"
          type={[Function]}
        />
        <div
          className="c6"
          direction="right"
        >
          <div
            className="c7"
          >
            2018-01-02
             
          </div>
          <div
            className="c15"
          >
            description222
             
          </div>
        </div>
      </div>
      <div
        className="c13"
        type={[Function]}
      >
        <div
          className="c3"
        >
          <div
            className="c4"
            type={[Function]}
          />
        </div>
        <div
          className="c14"
          type={[Function]}
        />
        <div
          className="c6"
          direction="right"
        >
          <div
            className="c7"
          >
            2018-01-03
             
          </div>
          <div
            className="c15"
          >
            description333
             
          </div>
        </div>
      </div>
      <div
        className="c13"
        type={[Function]}
      >
        <div
          className="c3"
        >
          <div
            className="c4"
            type={[Function]}
          />
        </div>
        <div
          className="c14"
          type={[Function]}
        />
        <div
          className="c6"
          direction="right"
        >
          <div
            className="c7"
          >
            2018-01-04
             
          </div>
          <div
            className="c15"
          >
            description444
             
          </div>
        </div>
      </div>
      <div
        className="c13"
        type={[Function]}
      >
        <div
          className="c3"
        >
          <div
            className="c4"
            type={[Function]}
          />
        </div>
        <div
          className="c16"
          type={[Function]}
        />
        <div
          className="c6"
          direction="right"
        >
          <div
            className="c7"
          >
            2018-01-05
             
          </div>
          <div
            className="c15"
          >
            description555
             
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="c0"
  >
    <p>
      左右分散排列 简洁样式
    </p>
    <br />
    <div
      className="c1"
    >
      <div
        className="c8"
        type={[Function]}
      >
        <div
          className="c3"
        >
          <div
            className="c4"
            type={[Function]}
          />
        </div>
        <div
          className="c9"
          type={[Function]}
        />
        <div
          className="c17"
          direction="left"
        >
          <div
            className="c7"
          >
            2018-01-01
             
          </div>
        </div>
      </div>
      <div
        className="c8"
        type={[Function]}
      >
        <div
          className="c3"
        >
          <div
            className="c4"
            type={[Function]}
          />
        </div>
        <div
          className="c9"
          type={[Function]}
        />
        <div
          className="c6"
          direction="right"
        >
          <div
            className="c7"
          >
            2018-01-02
             
          </div>
        </div>
      </div>
      <div
        className="c8"
        type={[Function]}
      >
        <div
          className="c3"
        >
          <div
            className="c4"
            type={[Function]}
          />
        </div>
        <div
          className="c9"
          type={[Function]}
        />
        <div
          className="c17"
          direction="left"
        >
          <div
            className="c7"
          >
            2018-01-03
             
          </div>
        </div>
      </div>
      <div
        className="c8"
        type={[Function]}
      >
        <div
          className="c3"
        >
          <div
            className="c4"
            type={[Function]}
          />
        </div>
        <div
          className="c9"
          type={[Function]}
        />
        <div
          className="c6"
          direction="right"
        >
          <div
            className="c7"
          >
            2018-01-04
             
          </div>
        </div>
      </div>
      <div
        className="c8"
        type={[Function]}
      >
        <div
          className="c3"
        >
          <div
            className="c4"
            type={[Function]}
          />
        </div>
        <div
          className="c12"
          type={[Function]}
        />
        <div
          className="c17"
          direction="left"
        >
          <div
            className="c7"
          >
            2018-01-05
             
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="c0"
  >
    <p>
       左右分散排列 有详情描述 简洁样式
    </p>
    <br />
    <div
      className="c1"
    >
      <div
        className="c13"
        type={[Function]}
      >
        <div
          className="c3"
        >
          <div
            className="c4"
            type={[Function]}
          />
        </div>
        <div
          className="c14"
          type={[Function]}
        />
        <div
          className="c17"
          direction="left"
        >
          <div
            className="c7"
          >
            2018-01-01
             
          </div>
          <div
            className="c15"
          >
            description111
             
          </div>
        </div>
      </div>
      <div
        className="c13"
        type={[Function]}
      >
        <div
          className="c3"
        >
          <div
            className="c4"
            type={[Function]}
          />
        </div>
        <div
          className="c14"
          type={[Function]}
        />
        <div
          className="c6"
          direction="right"
        >
          <div
            className="c7"
          >
            2018-01-02
             
          </div>
          <div
            className="c15"
          >
            description222
             
          </div>
        </div>
      </div>
      <div
        className="c13"
        type={[Function]}
      >
        <div
          className="c3"
        >
          <div
            className="c4"
            type={[Function]}
          />
        </div>
        <div
          className="c14"
          type={[Function]}
        />
        <div
          className="c17"
          direction="left"
        >
          <div
            className="c7"
          >
            2018-01-03
             
          </div>
          <div
            className="c15"
          >
            description333
             
          </div>
        </div>
      </div>
      <div
        className="c13"
        type={[Function]}
      >
        <div
          className="c3"
        >
          <div
            className="c4"
            type={[Function]}
          />
        </div>
        <div
          className="c14"
          type={[Function]}
        />
        <div
          className="c6"
          direction="right"
        >
          <div
            className="c7"
          >
            2018-01-04
             
          </div>
          <div
            className="c15"
          >
            description444
             
          </div>
        </div>
      </div>
      <div
        className="c13"
        type={[Function]}
      >
        <div
          className="c3"
        >
          <div
            className="c4"
            type={[Function]}
          />
        </div>
        <div
          className="c16"
          type={[Function]}
        />
        <div
          className="c17"
          direction="left"
        >
          <div
            className="c7"
          >
            2018-01-05
             
          </div>
          <div
            className="c15"
          >
            description555
             
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="c0"
  >
    <p>
      不同状态 简洁样式
    </p>
    <br />
    <div
      className="c1"
    >
      <div
        className="c8"
        type={[Function]}
      >
        <div
          className="c3"
        >
          <div
            className="c18"
            type={[Function]}
          />
        </div>
        <div
          className="c9"
          type={[Function]}
        />
        <div
          className="c6"
          direction="right"
        >
          <div
            className="c7"
          >
            2018-01-01
             
          </div>
        </div>
      </div>
      <div
        className="c8"
        type={[Function]}
      >
        <div
          className="c3"
        >
          <div
            className="c4"
            type={[Function]}
          />
        </div>
        <div
          className="c9"
          type={[Function]}
        />
        <div
          className="c6"
          direction="right"
        >
          <div
            className="c7"
          >
            2018-01-02
             
          </div>
        </div>
      </div>
      <div
        className="c8"
        type={[Function]}
      >
        <div
          className="c3"
        >
          <div
            className="c19"
            type={[Function]}
          />
        </div>
        <div
          className="c9"
          type={[Function]}
        />
        <div
          className="c6"
          direction="right"
        >
          <div
            className="c7"
          >
            2018-01-03
             
          </div>
        </div>
      </div>
      <div
        className="c8"
        type={[Function]}
      >
        <div
          className="c3"
        >
          <div
            className="c4"
            type={[Function]}
          />
        </div>
        <div
          className="c9"
          type={[Function]}
        />
        <div
          className="c6"
          direction="right"
        >
          <div
            className="c7"
          >
            2018-01-04
             
          </div>
        </div>
      </div>
      <div
        className="c8"
        type={[Function]}
      >
        <div
          className="c3"
        >
          <div
            className="c4"
            type={[Function]}
          />
        </div>
        <div
          className="c12"
          type={[Function]}
        />
        <div
          className="c6"
          direction="right"
        >
          <div
            className="c7"
          >
            2018-01-05
             
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="c0"
  >
    <p>
      自定义元素 图标样式 
    </p>
    <br />
    <div
      className="c1"
    >
      <div
        className="c13"
        type={[Function]}
      >
        <div
          className="c3"
        >
          <div
            className="c4"
            type={[Function]}
          />
        </div>
        <div
          className="c14"
          type={[Function]}
        />
        <div
          className="c6"
          direction="right"
        >
          <div
            className="c7"
          >
            2018-01-01
             
          </div>
          <div
            className="c15"
          >
            description111
             
          </div>
        </div>
      </div>
      <div
        className="c13"
        type={[Function]}
      >
        <div
          className="c3"
        >
          <div
            className="c4"
            type={[Function]}
          />
        </div>
        <div
          className="c14"
          type={[Function]}
        />
        <div
          className="c6"
          direction="right"
        >
          <div
            className="c7"
          >
            2018-01-02
             
          </div>
          <div
            className="c15"
          >
            description222
             
          </div>
        </div>
      </div>
      <div
        className="c13"
        type={[Function]}
      >
        <div
          className="c3"
        >
          <div
            className="c4"
            type={[Function]}
          />
        </div>
        <div
          className="c14"
          type={[Function]}
        />
        <div
          className="c6"
          direction="right"
        >
          <div
            className="c7"
          >
            2018-01-03
             
          </div>
          <div
            className="c15"
          >
            description333
             
          </div>
        </div>
      </div>
      <div
        className="c13"
        type={[Function]}
      >
        <div
          className="c3"
        >
          <div
            className="c4"
            type={[Function]}
          />
        </div>
        <div
          className="c14"
          type={[Function]}
        />
        <div
          className="c6"
          direction="right"
        >
          <div
            className="c7"
          >
            2018-01-04
             
          </div>
          <div
            className="c15"
          >
            description444
             
          </div>
        </div>
      </div>
      <div
        className="c13"
        type={[Function]}
      >
        <div
          className="c3"
        >
          <div
            className="c4"
            type={[Function]}
          />
        </div>
        <div
          className="c16"
          type={[Function]}
        />
        <div
          className="c6"
          direction="right"
        >
          <div
            className="c7"
          >
            2018-01-05
             
          </div>
          <div
            className="c15"
          >
            description555
             
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="c0"
  >
    <p>
       默认 幽灵节点样式 
    </p>
    <br />
    <div
      className="c1"
    >
      <div
        className="c13"
        type={[Function]}
      >
        <div
          className="c3"
        >
          <div
            className="c4"
            type={[Function]}
          />
        </div>
        <div
          className="c14"
          type={[Function]}
        />
        <div
          className="c6"
          direction="right"
        >
          <div
            className="c7"
          >
            2018-01-01
             
          </div>
          <div
            className="c15"
          >
            description111
             
          </div>
        </div>
      </div>
      <div
        className="c13"
        type={[Function]}
      >
        <div
          className="c3"
        >
          <div
            className="c4"
            type={[Function]}
          />
        </div>
        <div
          className="c14"
          type={[Function]}
        />
        <div
          className="c6"
          direction="right"
        >
          <div
            className="c7"
          >
            2018-01-02
             
          </div>
          <div
            className="c15"
          >
            description222
             
          </div>
        </div>
      </div>
      <div
        className="c13"
        type={[Function]}
      >
        <div
          className="c3"
        >
          <div
            className="c4"
            type={[Function]}
          />
        </div>
        <div
          className="c14"
          type={[Function]}
        />
        <div
          className="c6"
          direction="right"
        >
          <div
            className="c7"
          >
            2018-01-03
             
          </div>
          <div
            className="c15"
          >
            description333
             
          </div>
        </div>
      </div>
      <div
        className="c13"
        type={[Function]}
      >
        <div
          className="c3"
        >
          <div
            className="c4"
            type={[Function]}
          />
        </div>
        <div
          className="c14"
          type={[Function]}
        />
        <div
          className="c6"
          direction="right"
        >
          <div
            className="c7"
          >
            2018-01-04
             
          </div>
          <div
            className="c15"
          >
            description444
             
          </div>
        </div>
      </div>
      <div
        className="c13"
        type={[Function]}
      >
        <div
          className="c3"
        >
          <i
            className="lugia-icon-financial_loading_o c20 c21"
          />
        </div>
        <div
          className="c16"
          type={[Function]}
        />
        <div
          className="c6"
          direction="right"
        >
          <div
            className="c7"
          >
            2018-01-05
             
          </div>
          <div
            className="c15"
          >
            description555
             
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="c0"
  >
    <p>
      自定义 幽灵节点内容为 Icon 
    </p>
    <br />
    <div
      className="c1"
    >
      <div
        className="c13"
        type={[Function]}
      >
        <div
          className="c3"
        >
          <div
            className="c4"
            type={[Function]}
          />
        </div>
        <div
          className="c14"
          type={[Function]}
        />
        <div
          className="c6"
          direction="right"
        >
          <div
            className="c7"
          >
            2018-01-01
             
          </div>
          <div
            className="c15"
          >
            description111
             
          </div>
        </div>
      </div>
      <div
        className="c13"
        type={[Function]}
      >
        <div
          className="c3"
        >
          <div
            className="c4"
            type={[Function]}
          />
        </div>
        <div
          className="c14"
          type={[Function]}
        />
        <div
          className="c6"
          direction="right"
        >
          <div
            className="c7"
          >
            2018-01-02
             
          </div>
          <div
            className="c15"
          >
            description222
             
          </div>
        </div>
      </div>
      <div
        className="c13"
        type={[Function]}
      >
        <div
          className="c3"
        >
          <div
            className="c4"
            type={[Function]}
          />
        </div>
        <div
          className="c14"
          type={[Function]}
        />
        <div
          className="c6"
          direction="right"
        >
          <div
            className="c7"
          >
            2018-01-03
             
          </div>
          <div
            className="c15"
          >
            description333
             
          </div>
        </div>
      </div>
      <div
        className="c13"
        type={[Function]}
      >
        <div
          className="c3"
        >
          <div
            className="c4"
            type={[Function]}
          />
        </div>
        <div
          className="c14"
          type={[Function]}
        />
        <div
          className="c6"
          direction="right"
        >
          <div
            className="c7"
          >
            2018-01-04
             
          </div>
          <div
            className="c15"
          >
            description444
             
          </div>
        </div>
      </div>
      <div
        className="c13"
        type={[Function]}
      >
        <div
          className="c3"
        >
          <i
            className="lugia-icon-financial_abort  c21"
          />
        </div>
        <div
          className="c16"
          type={[Function]}
        />
        <div
          className="c6"
          direction="right"
        >
          <div
            className="c7"
          >
            2018-01-05
             
          </div>
          <div
            className="c15"
          >
            description555
             
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    style={
      Object {
        "margin": 30,
      }
    }
  >
    <button
      className="c22"
      onClick={[Function]}
      onMouseDown={[Function]}
      onMouseEnter={[Function]}
      onMouseOut={[Function]}
      onMouseOver={[Function]}
      onMouseUp={[Function]}
      size="default"
      type="primary"
    >
      <span
        className="c23"
        size="default"
        type="primary"
      >
        点击反转
      </span>
    </button>
    <div
      className="c1"
    >
      <div
        className="c13"
        type={[Function]}
      >
        <div
          className="c3"
        >
          <div
            className="c4"
            type={[Function]}
          />
        </div>
        <div
          className="c14"
          type={[Function]}
        />
        <div
          className="c6"
          direction="right"
        >
          <div
            className="c7"
          >
            2018-01-01
             
          </div>
          <div
            className="c15"
          >
            description111
             
          </div>
        </div>
      </div>
      <div
        className="c13"
        type={[Function]}
      >
        <div
          className="c3"
        >
          <div
            className="c4"
            type={[Function]}
          />
        </div>
        <div
          className="c14"
          type={[Function]}
        />
        <div
          className="c6"
          direction="right"
        >
          <div
            className="c7"
          >
            2018-01-02
             
          </div>
          <div
            className="c15"
          >
            description222
             
          </div>
        </div>
      </div>
      <div
        className="c13"
        type={[Function]}
      >
        <div
          className="c3"
        >
          <div
            className="c4"
            type={[Function]}
          />
        </div>
        <div
          className="c14"
          type={[Function]}
        />
        <div
          className="c6"
          direction="right"
        >
          <div
            className="c7"
          >
            2018-01-03
             
          </div>
          <div
            className="c15"
          >
            description333
             
          </div>
        </div>
      </div>
      <div
        className="c13"
        type={[Function]}
      >
        <div
          className="c3"
        >
          <div
            className="c4"
            type={[Function]}
          />
        </div>
        <div
          className="c14"
          type={[Function]}
        />
        <div
          className="c6"
          direction="right"
        >
          <div
            className="c7"
          >
            2018-01-04
             
          </div>
          <div
            className="c15"
          >
            description444
             
          </div>
        </div>
      </div>
      <div
        className="c13"
        type={[Function]}
      >
        <div
          className="c3"
        >
          <div
            className="c4"
            type={[Function]}
          />
        </div>
        <div
          className="c16"
          type={[Function]}
        />
        <div
          className="c6"
          direction="right"
        >
          <div
            className="c7"
          >
            2018-01-05
             
          </div>
          <div
            className="c15"
          >
            description555
             
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
